README.md

```diff
@@ -43,4 +46,13 @@ The script `run_devel.sh` creates a devel container (or start/attach to an exist
 ```console
 ./run-devel.sh -d melodic
 ```
-will attach to the container `ros-melodic-devel`.
+will attach to the container `ros-melodic-devel`. 
+
+### Running sudo commands
+
+You may notice the default user `orise` does not have root privileges. If really desired, one may run sudo commands by executing it outside the container by using `docker exec`. So for instance to install some package called `some-dependency`, the user would run:
```

Comment:
I think it is not so clear, what about something like:

> By default, the `orise` user does not have root privileges. If you desire to run sudo commands, you can execute them from outside the container by using `docker exec`. For instance, to install the deb package `foo`, the user can run:

...

Reply:
> I think it is not so clear, what about something like:
> 
> > By default, the `orise` user does not have root privileges. If you desire to run sudo commands, you can execute them from outside the container by using `docker exec`. For instance, to install the deb package `foo`, the user can run:
> 
> ...

Changed ec85476